config/routes.rb

```diff
@@ -3,5 +3,11 @@
   devise_for :users
   root to: 'books#index'
   resources :books
-  resources :users, only: %i(index show)
+  resources :users, only: %i(index show) do
+    resources :follows, only: %i(create destroy)
+    member do
+      get 'followings'
+      get 'followers'
```

Comment:
これは `users` ではなく `followings` と `followers` という resource を扱う routes に見えるので、自分で実装するなら以下のようにする気がします。

```suggestion
      resources 'followings', only: :index
      resources 'followers', only: :index
```

参考: [DHHはどのようにRailsのコントローラを書くのか](https://postd.cc/how-dhh-organizes-his-rails-controllers/)

Reply:
なるほど、勉強になります。参考記事もありがとうございます🙏